nicsdru_origins_theme.theme

```diff
@@ -358,6 +393,41 @@ function nicsdru_origins_theme_form_system_theme_settings_alter(&$form, FormStat
     '#default_value' => theme_get_setting('social_links_rss'),
     '#placeholder' => 'Enter a URL, such as https://example.com/rss.xml',
   ];
+
+  // Settings to disable Google Analytics tracking.
+  if (\Drupal::moduleHandler()->moduleExists('google_analytics')) {
+
+    $ga_account = \Drupal::config('google_analytics.settings')->get('account');
+
+    if (!empty($ga_account)) {
+
+      $form['ga_tracking_options'] = [
+        '#type' => 'details',
+        '#title' => t('Google Analytics tracking'),
+        '#open' => TRUE,
+      ];
+
+      $form['ga_tracking_options']['ga_tracking_disabled'] = [
```

Comment:
Just wondering Steve why there would be a setting to disable GA tracking altogether?

Reply:
My thinking is allow a site to be completely configured with GA but with the ability to prevent tracking until it's live, then it's simply a case of turning on the EU cookie compliance option or unchecking the disable for all.

Reply:
Spot on makes sense